docs/continuous_integration.md

```diff
@@ -1,16 +1,34 @@
 # vHive CI guide
 
 ## Deploying GitHub self-hosted runners
-* Create Create GitHub personal access token for your repo with [these](https://github.com/myoung34/docker-github-actions-runner#create-github-personal-access-token) scopes.
+
+### Deploying integration tests workflow runners
+1. Create GitHub personal access token for your repo with [these](https://github.com/myoung34/docker-github-actions-runner#create-github-personal-access-token) scopes.
```

Comment:
```suggestion
1. Create GitHub personal access token (PAT) for your repo with [these](https://github.com/myoung34/docker-github-actions-runner#create-github-personal-access-token) scopes.
```

Reply:
Fixed.

Reply:
where? I don't see it. You use `GitHub PAT` below, hence you need to define it.